src/popup.html

```diff
@@ -21,7 +21,7 @@
     <div id="header"><h1>ReadLater</h1><span>Save now, Read Later</span></div>
     <!--<div id="currentLink"></div>-->
     <div id="container">
-      <span id="addBtn">ADD</span>
+      <span id="toggleBtn">TOGGLE</span>
```

Comment:
Is it possible to rename this button to ADD and REMOVE based on the presence of the URL in the storage?
This way the UI will make it more explicit to the user that the URL will be removed. 
Also, if the invocation is by keyboard shortcut then we should ask for user confirmation before item removal.